added the show commands in the docs (#2951)
hashicorp/web-unified-docs: content/terraform/v1.16.x (beta)/docs/cli/commands/stacks/deployment-group/show.mdx

---
page_title: terraform stacks deployment-group show command
description: The terraform stacks deployment-group show command shows details of a deployment group.
# START AUTO GENERATED METADATA, DO NOT EDIT
created_at: 2026-08-03T17:10:24.000Z
last_modified: 2026-08-03T17:10:24.000Z
# END AUTO GENERATED METADATA
---

# `terraform stacks deployment-group show` reference

Use the `terraform stacks deployment-group show` command to show details of a deployment group.

## Usage

```shell-session
$ terraform stacks <global-stacks-flags> deployment-group show <options>
```

## Description

The `terraform stacks deployment-group show` command retrieves relevant information for a given deployment group, including its deployment runs. 

> You can identify the target deployment group either by providing a deployment group ID directly, or by specifying the organization name, project name, stack name, and deployment group name. When name-based targeting is used, the deployment group associated with the latest configuration for the specified stack is targeted.

Reviewer: ```suggestion
You can target a deployment group by providing either its ID, or by specifying all of the following values associated with the configuration:

- organization name 
- project name
- Stack name
- deployment group name

When specifying names, Terraform shows the deployment group associated with the latest configuration for the Stack.
```
for consistency

## Options

- `-organization-name`: The name of the organization to target.

>     - Required when using name-based targeting.
>     - String data type.
>     - Overrides the ENV VAR `TF_STACKS_ORGANIZATION_NAME` if provided.

Reviewer: ```suggestion
    - Required when omitting `-deployment-group-id`.
    - String data type.
    - Overrides the `TF_STACKS_ORGANIZATION_NAME` environment variable.
```

- `-project-name`: The name of the project to target.

>     - Required when using name-based targeting.
>     - String data type.
>     - Overrides the ENV VAR `TF_STACKS_PROJECT_NAME` if provided.

Reviewer: ```suggestion
    - Required when omitting `-deployment-group-id`.
    - String data type.
    - Overrides the `TF_STACKS_PROJECT_NAME` environment variable.
```

> - `-stack-name`: The name of the stack to target.

Reviewer: ```suggestion
- `-stack-name`: The name of the Stack to target.
```
Know that it pains me to have to capitalize stack

>     - Required when using name-based targeting.
>     - String data type.
>     - Overrides the ENV VAR `TF_STACKS_STACK_NAME` if provided.

Reviewer: ```suggestion
    - Required omitting `-deployment-group-id`.
    - String data type.
    - Overrides the `TF_STACKS_STACK_NAME` environment variable.
```

> - `-deployment-group-name`: The name of the deployment group to show within the latest configuration of the specified stack.
>     - Required when using name-based targeting; requires `-organization-name`, `-project-name`, and `-stack-name`.
>     - String data type.
> - `-deployment-group-id`: The ID of the deployment group to show.
>     - Required when not using name-based targeting.
>     - String data type.
>     - Has precedence over `-deployment-group-name`.
> - `-json`: Output results in JSON format.
>     - Optional.
>     - Boolean flag.
>     - Default is false (human-readable output).

Reviewer: ```suggestion
- `-deployment-group-name`: The name of the deployment group to show. Terraform shows the group associated with the latest configuration of the specified Stack.
    - Required when omitting `-deployment-group-id`.
    - String data type.
- `-deployment-group-id`: The ID of the deployment group to show.
    - Required when omitting the [`-stack-name`](#stack-name), [`-project-name`](#project-name), [`-organization-name`](#organization-name), and [`-deployment-group-name`](#deployment-group-name) flags.
    - String data type.
    - When you provide both `-deployment-group-id` and the [`-stack-name`](#stack-name), [`-project-name`](#project-name), [`-organization-name`](#organization-name), and [`-deployment-group-name`](#deployment-group-name) flags, Terraform shows the specific group per the `-deployment-group-id` instead of the group associated with the latest Stack configuration returned by the other arguments.
- `-json`: Output results in JSON format. When set to `false`, Terraform formats the output as human-readable text.
    - Optional.
    - Boolean flag.
    - Default is `false`.
```
I assume that the precedence for `-deploymen-group-id` flags follows the same pattern as the stack ID flags?

## Examples

> The following command shows details of a deployment group using organization, project, stack, and deployment group names:

Reviewer: ```suggestion
The following command shows details of a deployment group using organization, project, Stack, and deployment group names:
```
